File: app/src/main/java/code/name/monkey/retromusic/preferences/AlbumCoverStylePreference.java
package code.name.monkey.retromusic.preferences;

import android.content.Context;
import android.util.AttributeSet;

import code.name.monkey.appthemehelper.common.prefs.supportv7.ATEDialogPreference;


public class AlbumCoverStylePreference extends ATEDialogPreference {
    public AlbumCoverStylePreference(Context context) {
        super(context);
    }

    public AlbumCoverStylePreference(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public AlbumCoverStylePreference(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public AlbumCoverStylePreference(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }
}

File: app/src/main/java/code/name/monkey/retromusic/preferences/AlbumCoverStylePreference.kt
package code.name.monkey.retromusic.preferences

import android.content.Context
import android.util.AttributeSet
import androidx.preference.DialogPreference


class AlbumCoverStylePreference : DialogPreference {
    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr)

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int, defStyleRes: Int) : super(context, attrs, defStyleAttr, defStyleRes)
}